x-pack/test/fleet_api_integration/apis/package_policy/upgrade.ts

```diff
@@ -226,7 +226,7 @@ export default function (providerContext: FtrProviderContext) {
       });
 
       describe('when "dryRun: true" is provided', function () {
-        it('should return a diff with missingVars', async function () {
+        it('should return a diff with no errors', async function () {
```

Comment:
I made the choice to no longer explicitly report deprecated/removed variables as part of the dry run/upgrade process. If this becomes something we want to track, it might make more sense to bake it into the `validatePackagePolicy` service method instead of in this API.